prisma/schema.prisma

```diff
@@ -212,3 +213,17 @@ model Label {
 
   @@map("Labels")
 }
+
+model ScrapImages {
+  id          String       @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  filePath    String       @unique
```

Comment:
We are frequently querying ScrapImages table using `where filePath = ?`, and filePath is unique. I suggest we add an index to this column for faster queries